### PR TITLE
Fix list removal

### PR DESCRIPTION
**Pull request by KPostOffice**
# Issue link

# What changes have been made

# Verification steps

## Checks
- [ ] I've made sure the tests are passing. 
- Testing Strategy
   - [ ] Unit tests
   - [ ] Manual tests
   - [ ] Testing is not required for this change

### CODE REVIEW

**Comment by KPostOffice** [comment]
Supercedes: https://github.com/project-codeflare/codeflare-sdk/pull/463

**Review by VanillaSpoon — COMMENTED** [review]
Ran though guided demos with this, everything appears to be mounted correctly and the tests are passing :) lgtm

File: tests/unit_test.py [hunk 1]
```diff
@@ -2740,6 +2756,9 @@ def test_enable_local_interactive(mocker):
         head_group_spec["template"]["spec"]["initContainers"][0]["volumeMounts"]
         == volume_mounts
     )
+    print(head_group_spec["template"]["spec"]["volumes"])
```
Comment (astefanutti) [inline tests/unit_test.py:2759]: nit: maybe remove the print statements.

**Review by astefanutti — APPROVED** [review]

**Review by Bobbins228 — APPROVED** [review]
/approve
Ran through basic and local interactive everything is mounted as it should be.

**Review by tedhtchang — COMMENTED** [review]
Is there instruction for the ODH users to generate the `odh-trusted-ca-bundle` configmap ?  otherwise /lgtm

File: tests/unit_test.py [hunk 2]
```diff
@@ -3074,13 +3090,14 @@ def test_rjc_list_jobs(ray_job_client, mocker):
...
+    # os.remove(f"{aw_dir}unit-test-cluster.yaml")
+    # os.remove(f"{aw_dir}prio-test-cluster.yaml")
+    # os.remove(f"{aw_dir}unit-test-default-cluster.yaml")
+    # os.remove(f"{aw_dir}test.yaml")
+    # os.remove(f"{aw_dir}raytest2.yaml")
+    # os.remove(f"{aw_dir}quicktest.yaml")
+    # os.remove("tls-cluster-namespace/ca.crt")
+    # os.remove("tls-cluster-namespace/tls.crt")
+    # os.remove("tls-cluster-namespace/tls.key")
+    # os.rmdir("tls-cluster-namespace")
```
Comment (tedhtchang) [inline tests/unit_test.py:3103]: Did you not want to cleanup ?

**Comment by astefanutti** [comment]
/retitle Mount ODH custom CA ConfigMap to default template

**Pull request merged**